d2core/d2map/d2mapengine/walk_mesh.go

```diff
@@ -51,8 +53,9 @@ func (m *MapEngine) RegenerateWalkPaths() {
 			index := subTileX + (subTileY * m.size.Width * 5)
```

Comment:
i think we should make `numSubtilesPerTile` a constant in d2common

Reply:
Yes it is. This PR is the minimum code needed to implement Position. There will be another PR at some point to actually remove all the magic numbers. Possibly some more Position implementation before that.

The purpose of Position is to handle the whole tile/subtile thing. Rather than use a constant here, `subTileX` and `subTileY` will be a `Position`. This way nothing except Position has to know about the tile to subtile ratio. Note, this PR also adds NewPositionTile which creates a position from tile coordinates, again using `Position.subTilesPerTile`.

Reply:
That makes way more sense than using a subtiles constant everywhere, i like it.